ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
Interesting, thanks for posting about this internally.

Does the iOS implementation actually call the `str` method? In my testing, this bug reproed on Android but not iOS.